src/profile/external-identities/NihAccount.ts

```diff
@@ -71,7 +71,7 @@ export const NihAccount = ({ nihToken }) => {
       Utils.cond(
         [!nihStatusLoaded, () => h(SpacedSpinner, ['Loading NIH account status...'])],
         [isLinking, () => h(SpacedSpinner, ['Linking NIH account...'])],
-        [!linkedNihUsername, () => div([h(ShibbolethLink, { button: true }, ['Log in to NIH'])])],
+        [!linkedNihUsername, () => div([h(ShibbolethLink, { button: true }, ['Log into NIH'])])],
```

Comment:
I believe "Log in to NIH" is grammatically correct.

https://www.scribbr.com/frequently-asked-questions/log-into-or-log-in-to/#:~:text=%E2%80%9CLog%20in%E2%80%9D%20is%20a%20phrasal,to%20update%20privacy%20settings%E2%80%9D).

https://helpcenter.veeam.com/docs/styleguide/tw/log_in_to.html